src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -149,6 +149,10 @@ export class CellModel extends Disposable implements ICellModel {
 		return this._attachments;
 	}
 
+	public set attachments(imageData: nb.ICellAttachments) {
+		this._attachments = imageData ?? {};
```

Comment:
imageData isn't really accurate - attachments can be many different things. Would suggest just using `attachments` as the param name.

And the type should be `nb.ICellAttachments | undefined`